base/broadcast.jl

```diff
@@ -1102,15 +1102,15 @@ struct BitMaskedBitArray{N,M}
     mask::BitArray{M}
     BitMaskedBitArray{N,M}(parent, mask) where {N,M} = new(parent, mask)
 end
-@inline function BitMaskedBitArray(parent::BitArray{N}, mask::BitArray{M}) where {N,M}
+function BitMaskedBitArray(parent::BitArray{N}, mask::BitArray{M}) where {N,M}
```

Comment:
```suggestion
@inline function BitMaskedBitArray(parent::BitArray{N}, mask::BitArray{M}) where {N,M}
```

This one was overzealous — we should keep the force `@inline` here for the`@boundscheck`.  It currently still inlines, but I think it's good practice to keep this here in case something else in here gets big enough to lose the auto-inline.